### ISSUE TITLE
supermag tests not downloading in unit tests.

### ISSUE THREAD

**Issue by jklenzing** [rstoneback/pysat#222]
The supermag tests in test_instruments.py are failing in Travis and locally.  If I try to manually download the test data in ipython, there is no problem.  Likely not a server issue, but a unit test issue.

```
======================================================================
FAIL: Checking load routine functionality for module with clean level "none":  supermag magnetometer indices 
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/rstoneback/pysat/pysat/tests/test_instruments.py", line 290, in check_load
    assert not inst.empty
AssertionError

======================================================================
FAIL: Checking load routine functionality for module with clean level "none":  supermag magnetometer  
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/rstoneback/pysat/pysat/tests/test_instruments.py", line 290, in check_load
    assert not inst.empty
AssertionError

======================================================================
FAIL: Checking load routine functionality for module with clean level "none":  supermag magnetometer all 
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/rstoneback/pysat/pysat/tests/test_instruments.py", line 290, in check_load
    assert not inst.empty
AssertionError
```

**Comment by jklenzing** [comment]
See logs in https://travis-ci.org/rstoneback/pysat/builds/537227030

**Comment by jklenzing** [comment]
Confirmed fails limited to clean_level="none"

**Comment by jklenzing** [comment]
requests is retrieving the text "securityerror" rather than a data file.  Not familiar enough with supermag to fix this.  @aburrell? @rstoneback?

**Comment by jklenzing** [comment]
> Confirmed fails limited to clean_level="none"

Followup: downloads work, but not loads.

**Comment by rstoneback** [comment]
Clean level none is the only clean level that pysat checks to make sure data is loaded. In this case I’m guessing nothing is working. I believe I got an email that he log in and download formats changed. They even had a note about excessive downloading from scripts. Perhaps that was us?

On May 26, 2019, at 8:00 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


Confirmed fails limited to clean_level="none"

Followup: downloads work, but not loads.
